hw/top_earlgrey/ip/xbar/doc/checklist.md

```diff
@@ -208,14 +203,14 @@ Review        | Signoff date                            | Done        | 2019-11-
 Documentation | [DESIGN_DELTAS_CAPTURED_IF_ANY][] | N/A         |
```

Comment:
with #912 landed, this milestone name changed. DESIGN_DELTAS_CAPTURED also changed (see #912)

Reply:
Rebased. Thanks!